test/testshared/directives.go

```diff
@@ -59,7 +59,13 @@ func ParseTestDirectives(tb testing.TB, sourcePath string) *RunContext {
 			continue
 		}
 
-		if !strings.HasPrefix(line, "//golangcitest:") {
+		switch {
+		case strings.HasPrefix(line, "//golangcitest:"):
+			// Ok
+		case !strings.Contains(line, "golangcitest"):
+			// Assume this is a regular comment (required for go-header tests)
+			continue
```

Comment:
```suggestion
```

this a directive it's not a regular comment.

Reply:
I was guessing the intent of the original statement (since there's no comment) is to check for typos such as a missing colon. Since the original statement makes it impossible to use TestFix to test go-header with `//` comments, I attempted to modify the condition to be more selective.

> this a directive it's not a regular comment.

If the comment is a properly formed directive, the first case will detect it. Otherwise, it needs to differentiate between a malformed directive and something that's not supposed to be a directive. I assume that any comment containing the string `golangcitest` is supposed to be a directive, so conversely I wrote the condition to ignore lines that *do not* contain that string.